helpers/validate/strings.go

```diff
@@ -43,5 +43,9 @@ func LowerCasedString(i interface{}, k string) ([]string, []error) {
 		return nil, []error{fmt.Errorf("%q cannot contain whitespace", k)}
 	}
 
+	if strings.ContainsAny(v, "@") {
```

Comment:
I initially went [through the Azure naming rules since _some_ services allow these iirc](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) thinking this was used more widely than it is - but since this functions only used for Private DNS, it'd probably be worth moving this into the Private DNS package at the same time, since that's the only usage at the moment:

![Screenshot 2021-08-19 at 09 21 41](https://user-images.githubusercontent.com/666005/130025579-09436329-3471-4f4e-af4a-0c431644a89f.png)

Reply:
Makes sense, thanks for the input, i've moved this validation to the Private DNS package.